src/script/abspath.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+#!/bin/sh
+if [ `uname` = Darwin ]; then
+    find `cd $1; pwd` -name $2 -print
```

Comment:
This is exactly the same as the one in line 8. Please merge it into the `#else` statement.